src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import hamburgerIcon from "../../assets/navbar/hamburger.svg";
+import logoPrimary from "../../assets/navbar/logoPrimary.svg";
+import logoSecondary from "../../assets/navbar/logoSecondary.svg";
+import Page from "./Page";
+import SideBarWrapper from "./SideBarWrapper";
+
+const Navbar = () => {
+  const [toggle, setToggle] = useState(false);
+  const page: Page = Page.MAIN;
+
+  let header = "JOHNJUD";
+  let color = "bg-primary";
+  let logo = logoPrimary;
+
+  switch (page) {
+    case Page.ADMIN:
+      header = "JOHNJUD แอดมิน";
+      color = "bg-secondary";
+      break;
+    case Page.FORTUNE:
+      color = "bg-johnjud-purple";
+      logo = logoSecondary;
+      break;
+    case Page.MAIN:
+      logo = "";
+      break;
+  }
```

Comment:
คือเราจะรู้ได้ยังไงว่า page นี้คือ page อะไร เช่นแบบสมมุติ user เข้าวาปมาที่หน้า fortune เลย มันก็ควรจะ set เป็นสีม่วงให้เลย

คิดว่าอันนี้ควรใช้ state มาเก็บ แล้วก็ check routes เอาว่าเป้น routes อะไร

Reply:
just a little improvement, it sound like a `props` / `context` rather than state? since state is intended for component's internal state. `props` / `context` is use to provide a shared-information thorough many components of the webpage.